cgroups/fs/apply_raw.go

```diff
@@ -161,6 +161,20 @@ func (m *Manager) Set(container *configs.Config) error {
 	return nil
 }
 
+func (m *Manager) AddProcess(pid int) error {
+	for name, path := range m.Paths {
+		_, ok := subsystems[name]
```

Comment:
`join` also creates the cgroup. Arguably this doesn't need to happen for Add. But it can also be argued that since the directory exists it will work as well. WDYT about making `join` call add?